### PR TITLE
PLATFORM-6436 upgrade to v0.8

### PR DESCRIPTION
**Pull request by harnash**
Bump kratos version to 0.8.x

### CODE REVIEW

File: driver/registry_default.go [hunk 1]
```diff
@@ -216,20 +227,18 @@ func (m *RegistryDefault) HealthHandler(_ context.Context) *healthx.Handler {
...
+					if err != nil {
+						return err
+					}
+
+					if status.HasPending() {
+						return errors.Errorf("migrations have not yet been fully applied")
+					}
+
+					return nil
+				},
```
Comment (abador) [inline driver/registry_default.go:241]: shouldn't this still be disabled?

File: identity/identity.go [hunk 2]
```diff
@@ -40,6 +47,13 @@ func (lt State) IsValid() error {
 	return errors.New("identity state is not valid")
 }
 
+//type IdentifierCredential struct {
+//	Subject      string `json:"subject"`
+//	Provider     string `json:"provider"`
+//	AccessToken  string `json:"access_token"`
+//	RefreshToken string `json:"refresh_token"`
+//}
+
```
Comment (abador) [inline identity/identity.go:56]: why is this commented out?

File: identity/identity.go [hunk 3]
```diff
@@ -60,6 +74,9 @@ type Identity struct {
 	// Credentials represents all credentials that can be used for authenticating this identity.
 	Credentials map[CredentialsType]Credentials `json:"credentials,omitempty" faker:"-" db:"-"`
 
+	//// IdentifierCredentials contains the access and refresh token for oidc identifier
+	//IdentifierCredentials []IdentifierCredential `json:"identifier_credentials,omitempty" faker:"-" db:"-"`
+
```
Comment (abador) [inline identity/identity.go:79]: also this. please add comments

**Review by abador — CHANGES_REQUESTED** [review]
Few small comments. I think that this will be very tricky to deploy on prod. Shouldn't we add a compatibility layer and switch the clients in a safer manner? I mostly mean the session endpoint that has a different URL,

File: driver/registry_default.go (hunk 1 above)
Comment (harnash) [reply driver/registry_default.go:241]: I believe we should be OK now after switching to ProxySQL. We can re-disable this when we notice issues (it should be visible straight after the first deploy).

File: identity/identity.go (hunk 2 above)
Comment (harnash) [reply identity/identity.go:56]: Upstream's work in progress I guess https://github.com/ory/kratos/blob/master/identity/identity.go#L50-L55

File: identity/identity.go (hunk 3 above)
Comment (harnash) [reply identity/identity.go:79]: The same. It is upstream :-)

**Review by abador — APPROVED** [review]

**Pull request merged**